AnnoyancesFilter/sections/popups.txt

```diff
@@ -2855,7 +2857,7 @@ hackernoon.com##.js-stickyFooter
 bostonglobe.com#$#body { overflow:visible!important; }
 bostonglobe.com#$#.meter-social-connect { display:none!important; }
 spravochnick.ru#$#body { overflow: visible!important; }
-spravochnick.ru##.m-popup
+press.princeton.edu,spravochnick.ru##.m-popup
```

Comment:
All rules will have to be written using css property. if we have issue with page scrolling, otherwise you'll break the site on content blockers. I'll rewrite those rules.